mono/utils/mono-rand.c

```diff
@@ -53,18 +53,18 @@ static gint file = -1;
    - Return -1 on error
    getrandom() is retried if it failed with EINTR: interrupted by a signal. */
 static gint
-mono_getrandom (guchar *buffer, gint buffer_size, gint flags, MonoError *error)
+mono_getrandom (guchar *buffer, intptr_t buffer_size, gint flags, MonoError *error)
 {
 	static gint getrandom_works = 1;
 	gint count = 0;
 	gint err;
 
+	error_init (error);
```

Comment:
Remove this `error_init`. `mono_getrandom` is only called by `mono_rand_try_get_bytes` which already calls `error_init`.